WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TitleAndValueRow.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+struct TitleAndValueRow: View {
+    let title: String
+    let value: String
+    let selectable: Bool
+    var action: () -> Void
+
+    var body: some View {
+        HStack {
+            Text(title).font(.body)
+            Spacer()
+            Text(value).font(.body).foregroundColor(Color(.textSubtle))
+
+            if selectable {
+                Image(uiImage: .chevronImage)
+                    .frame(width: 22.0, height: 22.0)
+                    .foregroundColor(Color(UIColor.gray(.shade30)))
+            }
+        }
+        .onTapGesture {
+            guard selectable else {
+                return
+            }
+            action()
+         }
+        .frame(height: 44)
+        .padding([.leading, .trailing], 16)
```

Comment:
Same question about having these magic numbers in enums. Just pointing them out in case we decide to change it, we don't miss them.